src/main/java/uk/ac/ebi/ampt2d/storage/FileSystemStorageService.java

```diff
@@ -0,0 +1,82 @@
+package uk.ac.ebi.ampt2d.storage;
+
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.UrlResource;
+import org.springframework.stereotype.Service;
+import org.springframework.util.FileSystemUtils;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+@Service
+public class FileSystemStorageService implements StorageService {
+    private Logger logger = Logger.getLogger(FileSystemStorageService.class);
+
+    private final Path rootLocation;
+
+    @Autowired
```

Comment:
No longer needed since Spring 4.3 -> [Spring 4.3 announcement](https://spring.io/blog/2016/03/04/core-container-refinements-in-spring-framework-4-3)